### PR TITLE
Issue #698:  Removed Secured Mission from upstream docs

### LINKED ISSUE #698: remove SSO mission from docs for now

### PR DESCRIPTION
**Pull request by inoxx03**
fixes #698 

@rhoads-zach mind taking a look?

### CODE REVIEW

File: docs/getting-started/master.adoc [hunk 1]
```diff
@@ -8,12 +8,12 @@ include::topics/templates/document-attributes.adoc[]
...
 
 [.lead]
-This guide takes you through modern application development with {ProductName}. It shows you how to create and deploy example cloud-native applications called Boosters to OpenShift using the {launcher} application. These example applications can serve as the foundation for your actual cloud-native microservices since they can be updated and redeployed using the same deployment process detailed in this guide. 
+This guide takes you through modern application development with {ProductName}. It shows you how to create and deploy example cloud-native applications called Boosters to OpenShift using the {launcher} application. These example applications can serve as the foundation for your actual cloud-native microservices since they can be updated and redeployed using the same deployment process detailed in this guide.
 
 
 include::topics/concept-getting-started-intro.adoc[leveloffset=+1]
 
-include::topics/proc-configure-launcher-sso-realm.adoc[leveloffset=+1]
+//include::topics/proc-configure-launcher-sso-realm.adoc[leveloffset=+1]
```
Comment (rhoads-zach) [inline docs/getting-started/master.adoc]: @inoxx03 this is unrelated to the SSO mission. This is about the Federated Identity access for the launcher. Do you mind uncommenting this line?

Comment (inoxx03) [reply docs/getting-started/master.adoc]: @rhoads-zach done, I uncommented the procedure topic.

**Comment by rhoads-zach** [comment]
merged: abeb8e1a10545a79e62f97dfa614b8bea362b88f
